website/docs/d/eks_addon_version.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "EKS"
+layout: "aws"
+page_title: "AWS: aws_eks_addon_version"
+description: |-
+  Retrieve information about versions of an EKS add-on
+---
+
+# Data Source: aws_eks_addon_version
+
+Retrieve information about EKS add-on versions supported across EKS cluster versions.
+
+## Example Usage
+
+```terraform
+data "aws_eks_addon_version" "default" {
```

Comment:
If adding filtering, would like to see some more examples.

Reply:
Please see https://github.com/hashicorp/terraform-provider-aws/pull/23157#discussion_r808586101